src/_pytest/compat.py

```diff
@@ -26,6 +26,12 @@
 )
 
 
+if sys.version_info >= (3, 8):
+    from importlib import metadata as importlib_metadata  # noqa
+else:
+    import importlib_metadata  # noqa
```

Comment:
not sure if I should use something more specific. I usually just set __all__ for "import proxies".

Reply:
That's fine. 👍